frontend/src/util/activity.js

```diff
@@ -103,12 +113,15 @@ const groupMessageListByActivityId = (messageList) => {
   return activityMap
 }
 
-const createActivityListFromActivityMap = async (activityMap, apiUrl) => {
```

Comment:
Same remark: for me declaring the function `async` has the advantage to make explicit that `await`/`.then()` is required to get the returned value.

Reply:
okay, I reverted all these changes